pkg/gopass/secrets/akv.go

```diff
@@ -260,15 +260,13 @@ func ParseAKV(in []byte) *AKV {
 			continue
 		}
 
-		line = strings.TrimSpace(line)
-
 		key, val, found := strings.Cut(line, kvSep)
 		if !found {
 			continue
 		}
 
 		key = strings.TrimSpace(key)
-		val = strings.TrimSpace(val)
```

Comment:
Should I add a more detailed comment of why val wasn't `Trim`med? It would seem arbitrary otherwise.

Do  you normally reference GitHub issues on comments?

Reply:
Remove the dead code and add a reference to the GH Issue, please.